src/track/trackmetadatataglib.cpp

```diff
@@ -838,6 +838,29 @@ void readTrackMetadataFromID3v2Tag(TrackMetadata* pTrackMetadata,
         parseTrackPeak(pTrackMetadata,
                 toQString(pTrackPeakFrame->fieldList()[1]));
     }
+    TagLib::ID3v2::FrameList ratingFrame = tag.frameListMap()["POPM"];
+    int rating = 0;
+    if(!ratingFrame.isEmpty()) {
+        // RatingString "traktor@native-instruments.de rating=255 counter=2"
+
+        QString sRating = TStringToQString(ratingFrame.front()->toString());
+        sRating = sRating.section("=",1,2).left(3);
+        float fRating = sRating.toInt();
+        rating = ceil(fRating/51);
```

Comment:
With you idea of not connecting ratings with stars automatically, but only if flagged, we'd need to have a new column in the track table called iRating or similar? this wouldn't work so well though, as searching for rating "rating:>3" would pull mixxx's internal only star rating, wouldn't pick up the id3 rating.